apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -1184,45 +1194,51 @@ export class CaseService {
 
   private addMessagesForCompletedIndictmentCaseToQueue(
     theCase: Case,
+    updatedCase: Case,
     user: TUser,
   ): Promise<void> {
-    const messages: Message[] =
-      theCase.caseFiles
-        ?.filter(
-          (caseFile) =>
-            caseFile.state === CaseFileState.STORED_IN_RVG &&
-            caseFile.key &&
-            caseFile.category &&
-            [CaseFileCategory.COURT_RECORD, CaseFileCategory.RULING].includes(
-              caseFile.category,
-            ),
-        )
-        .map((caseFile) => ({
+    const messages: Message[] = []
+
+    // No need to deliver case files if completing after correction
```

Comment:
But won't we deliver for example the updated ruling when correction is completed?

Reply:
Or do we trigger that update somewhere else? For example when submitting a correction